docs/modules/operation/pages/deep-dive/visualizations/opsboard/dashlets/summary.adoc

```diff
@@ -10,7 +10,7 @@ The Summary dashlet shows a trend of incoming alarms in a given time frame in th
 | Field | Description
 
 | Boost support
-| Change priority of the dashlet on an ops board, depending on alarm level severity (see xref:deep-dive/admin/webui/opsboard/boosting-behavior.adoc[Boosted Severity]).
+| Change priority of the dashlet on an ops board, depending on alarm level severity (see xref:deep-dive/visualizations/opsboard/boosting-behavior.adoc[Boosted Severity]).
```

Comment:
```suggestion
| Change priority of the dashlet on an ops board, depending on alarm level severity (see xref:deep-dive/visualizations/opsboard/boosting-behavior.adoc[Boosted severity]).
```
to be consistent elsewhere, but since this isn't the actual title of the section, maybe they all should be lower case? (boosted severity)

Reply:
I think we could just let Antora pick up the title of the page for this one. :)

```suggestion
| Change priority of the dashlet on an ops board, depending on alarm level severity (see xref:deep-dive/visualizations/opsboard/boosting-behavior.adoc[]).
```